test/libsolidity/semanticTests/viaYul/dirty_memory_read_write_bytes4.sol

```diff
@@ -0,0 +1,21 @@
+contract C {
+    function f() public pure returns (bool correct) {
+        uint256[1] memory m;
+        // Emulate dirty bit
+        m[0] = 1 << 255;
```

Comment:
Why does this emulate the dirty bit, since it fits in `uint256`?

Reply:
I think the comment is misleading, I meant to test whether higher order bit is correctly cleared after the variable is reassigned.

Reply:
But shouldn't it be 1<<256 then?

Reply:
Dirty bit relative to the value that supersedes it. Since the overwritten value is something like `deadbeef` the emulated dirty bit is any bit of a higher order than the highest order set bit in `deadbeef`.

Reply:
If you use an inline-assembly-mstore as the next instruction, why is this line needed?

Reply:
Updated.